airflow/api_connexion/endpoints/forward_to_fab_endpoint.py

```diff
@@ -28,6 +30,7 @@
 
     from airflow.api_connexion.types import APIResponse
 
+pytestmark = pytest.mark.db_test
```

Comment:
I think this is wrong - Pytest + mark_db_test should not be in "airflow" code - only in test code.

Reply:
oops. Thought I added this in test code :). My pre-commit has been failing to run with invalid manifest error.
```
An error has occurred: InvalidManifestError: 
==> File /Users/ephraimbuddy/.cache/pre-commit/repo5pan_o01/.pre-commit-hooks.yaml
==> At Hook(id='forbid-crlf')
==> At key: stages
==> At index 0
=====> Expected one of commit, commit-msg, manual, merge-commit, post-checkout, post-commit, post-merge, post-rewrite, prepare-commit-msg, push but got: 'pre-commit'
Check the log at /Users/ephraimbuddy/.cache/pre-commit/pre-commit.log
```
So I skipped pre-commit which could have caught this.
The pre-commit has been resolved. My mac had version 2.20 so I install the latest and cleaned the cache. Now is working. Will fix this

Reply:
> oops. Thought I added this in test code :). My pre-commit has been failing to run with invalid manifest error.
> 
> ```
> An error has occurred: InvalidManifestError: 
> ==> File /Users/ephraimbuddy/.cache/pre-commit/repo5pan_o01/.pre-commit-hooks.yaml
> ==> At Hook(id='forbid-crlf')
> ==> At key: stages
> ==> At index 0
> =====> Expected one of commit, commit-msg, manual, merge-commit, post-checkout, post-commit, post-merge, post-rewrite, prepare-commit-msg, push but got: 'pre-commit'
> Check the log at /Users/ephraimbuddy/.cache/pre-commit/pre-commit.log
> ```
> 
> So I skipped pre-commit which could have caught this. The pre-commit has been resolved. My mac had version 2.20 so I install the latest and cleaned the cache. Now is working. Will fix this

Interesting! I get the exact same error but installing the new version did not fix it. I installed the last version (3.5) and ran `pre-commit clean`. Still got the same error though

Reply:
You can also manually remove all the pre-commit cache. it iis in `~/.cache/pre-commit` I believe. 

You know `Cache invalidation is ....`

Reply:
Yeah I tried but then after it complained it could not find X repo in cache ...

Reply:
Yeah, invalidating the cache is another thing I did but honestly, I don't know how it got resolved because last night, I was still having errors, then in the morning, everything worked fine. You can try restarting if everything fails

Reply:
I get this error only when I run static-checks through Breeze. From [this](https://github.com/Lucas-C/pre-commit-hooks/issues/83), it seems there was a breaking change in pre-commit and they recommend using the latest version of `pre-commit`. Do you know which version of `pre-commit` is picked by Breeze @potiuk ?

Reply:
The one that is available on path

Reply:
This: 

```
command_to_execute = [sys.executable, "-m", "pre_commit", "run"]
```

is the command so basically this is the pre-commit installed with breeze when you installed it with `pipx`. You might want to force-reinstall it with:

```
pipx upgrade apache-airflow-breeze
```

But we might also force upgrade for everyone to latest version of pre-commit. Let me just do it

Reply:
PR here: https://github.com/apache/airflow/pull/35510

Once we merge and you checkout the latest version and run breeze, breeze should reinstall itself with `pipx` automatically.